includes/Classifai/Admin/Metabox.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/**
+ * Metabox class.
+ *
+ * @since 2.2.3
+ * @package Classifai
+ */
+
+namespace Classifai\Admin;
+
+use Classifai\Providers\OpenAI\Embeddings;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * This class generates the metabox for the classic editor post types.
+ */
+class Metabox {
```

Comment:
So just my opinion here but because all the code here right now is specific to the Embeddings feature, I'd suggest we move this into that file rather than create a new class

Reply:
That said, if we want a more generic class, I think we'll want to migrate any existing meta box handling to use this. So up to you on how you want to approach that, just want to make sure we're consistent

Reply:
It makes sense. I moved the code within the Embeddings feature. Thanks, @dkotter!